styles/pages/learn-to-export.scss

```
@@ -336,3 +336,68 @@ li .lte-suptopic {
 .learn-card-description p {
   @extend .govuk-body
 }
+
+.related-links-cta-container {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  // horizontal scrolling of child elements
+  @include govuk-media-query($until: tablet) {
```

I think this bp needs to be `desktop` to match the bp where we lose the side bar.
```suggestion
  @include govuk-media-query($until: desktop) {
```